### PR TITLE
docs(tutorial): review

### PR DESCRIPTION
**Pull request by fopsdev**

### CODE REVIEW

**Comment by fopsdev** [comment]
only issue now with the tutorial i've found:
https://github.com/cerebral/cerebral/issues/771

File: docs/developer_guide/compute.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ import starsCount from './starsCount'
       showToast(string`The repos have ${starsCount} stars`, 5000, 'success')
     ],
     catch: new Map([
-      [HttpProviderError, showToast(string`Error: ${props`error.message`}`, 5000)]
+      [HttpProviderError, showToast(string`Error: ${props`error.body.message`}`, 5000)]
```
Comment (henri-hulski) [inline docs/developer_guide/compute.md]: `error.message` should be correct. 
See https://github.com/cerebral/cerebral/blob/master/packages/cerebral-provider-http/src/HttpProviderError.js

Comment (fopsdev) [inline docs/developer_guide/compute.md]: @henri didn't work for me when running it (from the BIN)
what i did:
i changed the repo to a name that doen't exist... then i got: Error: undefined
after changing it to .body.error it was fine...
maybe a different version of http-provider on webpackbin?

Comment (fopsdev) [inline docs/developer_guide/compute.md]: you can test it via the bin which is used in the "Router" -part:
https://www.webpackbin.com/bins/-KdBeIDJoRv0PQlF7uWU

but it is wrong anyway...so there is a bin change :)

Comment (henri-hulski) [inline docs/developer_guide/compute.md]: @fopsdev Yeah you're right.
No idea why.

**Pull request merged**